### ISSUE TITLE
Date Converter add copy and paste

### ISSUE THREAD

**Issue by dpdzq** [DevToys-app/DevToys#1468]
### What improvement do you think would an existing feature or tool in DevToys?

In Date Converter, I hope can add copy and paste the timestamp.  Sometimes we need to converter the "YYYY-mm-dd" to the timestamp and copy the timestamp to use. Ths

### Solution/Idea

Add copy and past the timestamp
![Uploading 1732782641789.jpg…]()

**Comment by yuvashrikarunakaran** [comment]
1. Add a "Copy Result" Button
After converting the date to a timestamp or vice versa, a simple "Copy" button next to the result would allow users to instantly copy the converted value.
This eliminates the need for manual selection and copying, saving time and reducing the chance of errors.
2. Paste Functionality for Input
Allow users to paste dates or timestamps directly into the input field via a context menu or shortcut (e.g., Ctrl + V).
This would be especially helpful when dealing with dates or timestamps from external sources like logs, APIs, or databases.
3. Support Multiple Formats
Enable automatic recognition of common date formats (e.g., YYYY-mm-dd, MM/DD/YYYY, dd-MM-YYYY) without requiring manual format selection.
Additionally, allow users to choose the output format for the timestamp (e.g., seconds, milliseconds).
4. History and Quick Copy
Keep a small history of recently converted values for quick access.
Each entry could have its own "Copy" button, making it easier to work with multiple dates or timestamps.
5. Keyboard-Driven Workflow
Make the entire Date Converter tool operable via the keyboard:
Tab navigation between fields.
Enter to perform conversions.
Shortcut keys for copy/paste.

**Comment by dpdzq** [comment]
Thanks for your answer. But In the app, I want to copy the timestamp value, but I cannot find the "copy" button beside the timestamp value. I use macOS App Apple silicon(ARM64) version.
![1733380225284](https://github.com/user-attachments/assets/723871f8-f550-43d6-a315-ff2feb943d3f)

**Comment by BokuNoMaxi** [comment]
I am here for the exact same reason. 

In the older versions of devtoys you could copy the timestamp. After an update it isn't possible anymore to copy the timestamp, you can only paste it into this field. BUT you can copy the date below the timestamp. 

you can even copy the information above the timestamp... if the information is up there it would help too!

**Comment by dpdzq** [comment]
Yes, in fact, sometimes it is necessary to copy timestamps. I want to use the timestamp value, not the date value.
